app/src/main/res/values/strings.xml

```diff
@@ -83,15 +83,15 @@
 
     <string name="off_type">Off type</string>
     <string name="annual_leave">Annual leave (14.5 days)</string>
-    <string name="leave_for_marriage">Leave for marriage (3.0 days)</string>
-    <string name="leave_for_child_marriage">Leave for child\'s marriage (1.0 days)</string>
-    <string name="funeral_leave">Funeral leave (3.0 days)</string>
-    <string name="leave_for_care_of_sick_child">Leave for care of sick child (20.0 day)</string>
-    <string name="pregnancy_examination_leave">Pregnancy examination leave (5.0 day)</string>
-    <string name="sick_leave">Sick leave (60.0 days)</string>
-    <string name="miscarriage_leave">Miscarriage leave (50.0 days)</string>
-    <string name="maternity_leave">Maternity leave (180.0 days)</string>
-    <string name="wife_labor_leave">Wife’s labor leave (14.0 days)</string>
+    <string name="leave_for_marriage">Leave for marriage (%s days)</string>
+    <string name="leave_for_child_marriage">Leave for child\'s marriage (%s days)</string>
+    <string name="funeral_leave">Funeral leave</string>
```

Comment:
ERROR: [StringFormatInvalid] Format string '`funeral_leave`' is not a valid format string so it should not be passed to `String.format`
 If a string contains a '%' character, then the string may be a formatting string which will be passed to `String.format` from Java code to replace each '%' occurrence with specific values.

This lint warning checks for two related problems:
(1) Formatting strings that are invalid, meaning that `String.format` will throw exceptions at runtime when attempting to use the format string.
(2) Strings containing '%' that are not formatting strings getting passed to a `String.format` call. In this case the '%' will need to be escaped as '%%'.

NOTE: Not all Strings which look like formatting strings are intended for use by `String.format`; for example, they may contain date formats intended for `android.text.format.Time#format()`. Lint cannot always figure out that a String is a date format, so you may get false warnings in those scenarios. See the suppress help topic for information on how to suppress errors in that case.